audit/src/test/resources/audit.yml

```diff
@@ -20,6 +20,9 @@ auditOnError: false
 # log level; by default set to info
 logLevelIsError: false
 
+# the format for outputting the timestamp
```

Comment:
lets also document here what would happen if the format that user entered is invalid.

Reply:
feat/858-audit-format